src/pages/docs/octopus-rest-api/cli/octopus-package.mdx

```diff
@@ -18,7 +18,7 @@ Available Commands:
   help Help about any command
   list List packages
   nuget Package as NuPkg
-  upload upload one or more packages to Octopus Deploy
+  upload Upload one or more packages to Octopus Deploy
```

Comment:
This is probably based on an actual program's buggy output, so please consider this as a bug report.